src/GraphQL/Validation/DocumentValidator.cs

```diff
@@ -88,7 +88,7 @@ public class DocumentValidator : IDocumentValidator
 
                 if (!rules.Any())
                 {
-                    variables = context.GetVariableValues(schema, variableDefinitions, inputs); // can report errors even without rules enabled
+                    variables = context.GetVariableValues(schema, variableDefinitions, inputs ?? Inputs.Empty); // can report errors even without rules enabled
```

Comment:
Bug fix.  Without the fix it can cause a NRE if variables are specified in the document but null was provided to inputs.  Already correct in the `else` condition of the above `if (!rules.Any())`.

https://github.com/graphql-dotnet/graphql-dotnet/blob/d9b96431d9962d6f0d1dd1070da5e4546522e065/src/GraphQL/Validation/DocumentValidator.cs#L128

Reply:
Seems so.